docs/processes/release/tasks/legal/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Legalities
+parent: Release & Milestone Tasks
+grand_parent: Contributors
+nav_order: 70
+---
+
+<!--
+© 2021 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Legalities
+
+## 1. Scan for Copyright notices
+
+Check source and data files, especially newly contributed ones, to make sure the
+proper copyright notice is in place. For example,
+
+```
+© 2016 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html#License
+```
+
+Scan the source code to make sure that every file that was touched recently has
+the current year in the copyright statement. See the [ICU Copyright
+Scanner](../../../copyright-scan.md) page and follow the link to the scripts and
+the readme.
+
+Scan the source code to include third-party company names in copyright notices
+if necessary.
+
+~~Scan for text files that do not contain the word "Copyright": find_textfiles
+-nosvn -novc -noeclipse | xargs grep -i -L Copyright (See the find_textfiles
+Perl script attached to this page.) There are files without word "Copyright" in
+ICU source repository including some test data files (no comment syntax defined
+for these test data files), Unicode data files, tzcode source files and others.
+Review the output file list and determine if each of them should have ICU
+copyright statement or not.~~ ***The script [find_textfiles](find_textfiles)
```

Comment:
Script linked here (@yumaoka).

I didn't want to think about what to update as part of the migration PR. And I didn't want to poke into the perl script to see if there's anything interesting in it. :)

https://sites.google.com/site/icusite/processes/release/tasks/legal